src/js/apps/globals/app-frame_app.js

```diff
@@ -9,10 +11,56 @@ import PatientsMainApp from 'js/apps/patients/patients-main_app';
 
 import { AppNavView, AppNavCollectionView } from 'js/views/globals/app-nav/app-nav_views';
 
+const patientsNav = new Backbone.Collection([{
+  titleI18nKey: 'globals.appNav.patients.allPatients',
+  event: 'patients:all',
+  eventArgs: [],
+}]);
+
+const viewsNav = new Backbone.Collection([
+  {
+    titleI18nKey: 'globals.appNav.views.ownedByMe',
+    event: 'view',
+    eventArgs: ['owned-by-me'],
+  },
+  {
+    titleI18nKey: 'globals.appNav.views.myRole',
+    event: 'view',
+    eventArgs: ['actions-for-my-role'],
+  },
+  {
+    titleI18nKey: 'globals.appNav.views.newActions',
+    event: 'view',
+    eventArgs: ['new-actions'],
+  },
+  {
+    titleI18nKey: 'globals.appNav.views.updatedPastThree',
+    event: 'view',
+    eventArgs: ['updated-past-three-days'],
+  },
+  {
+    titleI18nKey: 'globals.appNav.views.doneLastThirty',
+    event: 'view',
+    eventArgs: ['done-last-thirty-days'],
+  },
+]);
+
 export default App.extend({
   channelName: 'nav',
   radioRequests: {
-    'select': 'select',
+    'select': 'onSelect',
+  },
+  onSelect(appName, event, eventArgs) {
+    const navMatch = viewsNav.find(model => this._navMatch(model, event, eventArgs))
```

Comment:
I think this might be cleaner if it was `this._navMatch(viewsNav, event, eventArgs) || ...`  then `_navMatch` can just be a `find(model => { return...`  makes `_navMatch` a little bit more complex, but this line is simpler.